content/docs/object-reference/build/conda.md

```diff
@@ -0,0 +1,101 @@
+# Conda Environments
+
+Create a conda environment using requirements gathered from a model.
+
+## Description
+
+`mlem` can also create conda environments based on requirements gathered from a
+model. This naturally extends the functionality of the
+[`requirements builder`](/doc/object-reference/build/requirements) and is also
+similar to the [`venv builder`](/doc/object-reference/build/venv).
+
+In addition to installing conda packages, `pip` based packages will also be
+installed in the `conda` environment.
```

Comment:
So, `mlem build conda` actually installs packages to the env without any prior work? Or you hide the "manually adding part" Mike was talking about?

Reply:
Oh, we still need to add manually unfortunately. This is a discussion I am awaiting a conclusion for.

But the crux is, after you add manually -- we can install both -- conda based and pip based packages in the conda environment.

Reply:
Got you. Then if we keep the current implementation, we should explain that one should add them manually. I posted my opinion in that ticket, so it's up to you guys to decide whether we want this implemented or not, and how.

Reply:
updated the docs on how to pass manually. Resolving...